scripts/README.md

```diff
@@ -8,3 +8,20 @@ For best experience with git scripts, create alias for each. Edit your `~/.gitco
 [alias]
 	add-heroku-remote = !${path_to_git_script}
 ```
+
+The exclamation mark before path is required.
+
+Scripts:
+
+* [git\_heroku\_add.sh](https://github.com/DVELP/cookbook/blob/master/scripts/git_heroku_add.sh)
```

Comment:
Can you show the full example with more than one script? It's not clear to me how it would look nice when there's more than one script:

Scripts:

Setup Heroku Remotes

Adds git remotes for heroku

[git\_heroku\_add.sh](https://github.com/DVELP/cookbook/blob/master/scripts/git_heroku_add.sh)

* Requires two parameters: remote_name and heroku_app_name
* Example: `git_heroku_add.sh staging dvelp-staging`


Setup Heroku Remotes

Adds git remotes for heroku

[git\_heroku\_add.sh](https://github.com/DVELP/cookbook/blob/master/scripts/git_heroku_add.sh)

* Requires two parameters: remote_name and heroku_app_name
* Example: `git_heroku_add.sh staging dvelp-staging`